web/src/stream_settings_ui.js

```diff
@@ -688,7 +689,15 @@ export function switch_to_stream_row(stream_id) {
 
     scroll_util.scroll_element_into_container($stream_row, $container);
 
-    stream_edit.open_edit_panel_for_row($stream_row);
+    stream_edit.open_edit_panel_for_row($stream_row, right_section);
+    const right_section_name = stream_edit.right_section_url_fragment_to_section_name.get(
+        right_section ?? "general",
```

Comment:
What's the thinking behind the `general` default here?

Reply:
Changed it so the caller always passes the correct value